config/locales/rules_of_origin.yml

```diff
@@ -0,0 +1,16 @@
+---
+en:
+  rules_of_origin:
+    preferential_treatment:
+      UA:
```

Comment:
this could be confusing since it is not the language used, but the county code :)

Reply:
So I think we should just put this in en.yml so we can easily handover localised strings for translation?

Reply:
I also think only the first key is the actual locale :tada: so it shouldn't be too confusing